### PR TITLE
#9 Task. Dzmitry Bryt

### PR DESCRIPTION
**Pull request by Daqueze21**
https://travis-ci.org/github/Daqueze21/js-assignments/builds/715474974

### CODE REVIEW

**Comment by FedorovD** [comment]
Please clear PR code not related to Task#9 and click "ready for review" when it will be finished. Thank you!

**Comment by FedorovD** [comment]
Remove these changes from Task1-4, there should only be a task#9

**Comment by Daqueze21** [comment]
> Remove these changes from Task1-4, there should only be a task#9

Remove solutions from tasks  1-4. My fault!

**Comment by Daqueze21** [comment]
Update files

File: task/09-functions-n-closures-tasks.js [hunk 1]
```diff
@@ -171,7 +188,11 @@ function partialUsingArguments(fn) {
  *   getId10() => 11
  */
 function getIdGeneratorFunction(startFrom) {
-    throw new Error('Not implemented');
+    let id = startFrom - 1;
+    return () => {
+        id += 1;
+        return id;
```
Comment (FedorovD) [inline task/09-functions-n-closures-tasks.js:194]: can be more compact
`return id++`

**Comment by FedorovD** [comment]
Closing the PR as accepted.Thanks!

**Pull request closed without merging**